spec.bs

```diff
@@ -732,7 +745,13 @@ following [=struct/items=]:
   :: a [=fencedframetype/fenced frame reporting map=]
 
   : <dfn>direct seller is seller</dfn>
-  :: a boolean
+  :: a [=boolean=], initially true
+
+  : <dfn>allowed reporting origins</dfn>
+  :: null or a [=list=] of [=origins=]
```

Comment:
nit: it would be good to briefly explain what this field means, and what null means for the field.

Reply:
or maybe just say "passed from protected audience interest group's allowed reporting origins", with link to https://github.com/WICG/turtledove/blob/main/spec.bs#L3201.

Reply:
Added "An origin must be present in this list to be the destination of a [=fencedframetype/destination URL event=] report."